workflows/tasks/sicer.wdl

```diff
@@ -51,26 +47,32 @@ task sicer {
             -g ~{gap_size} \
             -e ~{evalue}
 
-        #mv ~{basename(bedfile,'.bed')}-W~{window}-G~{gap_size}.scoreisland ~{outputname}-W~{window}-G~{gap_size}.scoreisland
-        #mv ~{basename(bedfile,'.bed')}-W~{window}-normalized.wig ~{outputname}-W~{window}-normalized.wig
-        #mv ~{basename(bedfile,'.bed')}-W~{window}-G~{gap_size}-islands-summary ~{outputname}-W~{window}-G~{gap_size}-islands-summary
-        #if [ -f "~{basename(bedfile,'.bed')}-W~{window}-G~{gap_size}-FDR0.01-island.bed" ]; then
-        #    mv ~{basename(bedfile,'.bed')}-W~{window}-G~{gap_size}-FDR0.01-island.bed ~{outputname}-W~{window}-G~{gap_size}-FDR0.01-island.bed
+        #mv ~{basename(bedfile, ".bed")}-W~{window}-G~{gap_size}.scoreisland ~{outputname}-W~{
+            window}-G~{gap_size}.scoreisland
+        #mv ~{basename(bedfile, ".bed")}-W~{window}-normalized.wig ~{outputname}-W~{window
+            }-normalized.wig
+        #mv ~{basename(bedfile, ".bed")}-W~{window}-G~{gap_size}-islands-summary ~{
+            outputname}-W~{window}-G~{gap_size}-islands-summary
+        #if [ -f "~{basename(bedfile, ".bed")}-W~{window}-G~{gap_size}-FDR0.01-island.bed" ]; then
+        #    mv ~{basename(bedfile, ".bed")}-W~{window}-G~{gap_size}-FDR0.01-island.bed ~{
+            outputname}-W~{window}-G~{gap_size}-FDR0.01-island.bed
```

Comment:
bad format

Reply:
How "should" this look? As stated above, the fact this is commented out is irrelevant. The line is too long, so we should line break it.

Reply:
It's the color-coding of the code that is throwing me off, which isn't a `format` issue. Since the second line in each case isn't greyed out, it's initially confusing to look at. I don't know if we can do something for the grammars in GitHub and VSCode to make it easier for users.

Reply:
Yea the highlighting can definitely be improved in quite a few places/ways